lib/rsolr/client.rb

```diff
@@ -63,6 +65,11 @@ def paginate page, per_page, path, opts = nil
     execute build_paginated_request(page, per_page, path, opts)
   end
   
+  # Sends a query to /select or the custom configured select handler.
+  def select opts={}
```

Comment:
I think supporting both an instance-level and method-level configuration would be idea